scripts/linters/pylint_extensions_test.py

```diff
@@ -2234,3 +2234,191 @@ def test_file_overview_at_end_of_file(self):
+                    class ClassName(Class):
+                        \"\"\"This is a docstring.\"\"\"
+
+                    def func():
+                        a = 1 + 2
+                """)
+        node_class_with_no_body.file = filename
+        node_class_with_no_body.path = filename
+
+        self.checker_test_object.checker.process_module(
+            node_class_with_no_body)
+
+        with self.checker_test_object.assertNoMessages():
+            temp_file.close()
+
+    def test_none_and_excessive_newline_below_class_docstring(self):
+        node_none_and_excessive_newline_below_class_docstring = (
+            astroid.scoped_nodes.Module(
+                name='test',
+                doc='Custom test'))
+        temp_file = tempfile.NamedTemporaryFile()
+        filename = temp_file.name
+
+        with python_utils.open_file(filename, 'w') as tmp:
+            tmp.write(
+                u"""
+                    class ClassName(Class):
+                        \"\"\"This is a docstring.\"\"\"
+
+                        a = 1 + 2
+
+                    class ClassName(Class):
+                        \"\"\"This is a docstring.\"\"\"
+                        a = 1 + 2
+
+                    class ClassName(Class):
+                        \"\"\"This is a docstring.\"\"\"
+
+
+                        a = 1 + 2
+                """)
+        node_none_and_excessive_newline_below_class_docstring.file = filename
+        node_none_and_excessive_newline_below_class_docstring.path = filename
+
+        self.checker_test_object.checker.process_module(
+            node_none_and_excessive_newline_below_class_docstring)
+
+        message_with_no_newline = testutils.Message(
+            msg_id='newline-below-class-docstring',
+            line=9)
+
+        message_with_excessive_newline = testutils.Message(
+            msg_id='newline-below-class-docstring',
+            line=15)
+
+        with self.checker_test_object.assertAddsMessages(
+            message_with_no_newline, message_with_excessive_newline):
+            temp_file.close()
+
+    def test_single_newline_below_class_docstring(self):
+        node_with_no_error_message = astroid.scoped_nodes.Module(
+            name='test',
+            doc='Custom test')
+        temp_file = tempfile.NamedTemporaryFile()
+        filename = temp_file.name
+
+        with python_utils.open_file(filename, 'w') as tmp:
+            tmp.write(
+                u"""
+                    class ClassName(Class):
+                        \"\"\"This is a multiline.
+                        docstring.
+                        \"\"\"
+
+                        a = 1 + 2
+                """)
+        node_with_no_error_message.file = filename
+        node_with_no_error_message.path = filename
+
+        self.checker_test_object.checker.process_module(
+            node_with_no_error_message)
+
+        with self.checker_test_object.assertNoMessages():
+            temp_file.close()
```

Comment:
- Add test for a decorated function just after class docstring
- Add test for inline comment jsut after class's docstring (This can be for explaianing a contants define below docstring of a class)

Reply:
Done.